save-agent/src/linuxX64Main/kotlin/com/saveourtool/save/agent/SaveAgent.kt

```diff
@@ -319,6 +319,7 @@ class SaveAgent(private val config: AgentConfiguration,
                         tr.resources.test.toString(),
                         pluginExecution.plugin,
                         config.id,
+                        config.name,
```

Comment:
Do we need agent to know its name? Or orchestrator can manage all data based on ID?